translations/translated/mudlet_en_US.ts

```diff
@@ -4,140 +4,90 @@
-with a new symbol to use for all the rooms:</numerusform>
-            <numerusform>The only used symbol is &quot;%1&quot; in one or
-more of the selected %n rooms, delete
-this to clear it from those rooms or replace
-with a new symbol to use for all the rooms:</numerusform>
-        </translation>
-    </message>
-    <message numerus="yes">
-        <source>Choose:
- • an existing symbol from the list below (sorted by most commonly used first)
- • enter one or more graphemes (&quot;visible characters&quot;) as a new symbol
- • enter a space to clear any existing symbols
-for all of the %n selected room(s):</source>
-        <comment>Use line feeds to format text into a reasonable rectangle if needed, %n is the number of rooms involved.</comment>
-        <translation type="vanished">
-            <numerusform>Choose:
- • an existing symbol from the list below (sorted by most commonly used first)
- • enter one or more graphemes (&quot;visible characters&quot;) as a new symbol
- • enter a space to clear any existing symbols
-for the selected room:</numerusform>
-            <numerusform>Choose:
- • an existing symbol from the list below (sorted by most commonly used first)
- • enter one or more graphemes (&quot;visible characters&quot;) as a new symbol
- • enter a space to clear any existing symbols
-for all of the %n selected rooms:</numerusform>
+        <location filename="../../src/T2DMap.cpp" line="1256"/>
+        <source>You have a map loaded (%n room(s)), but Mudlet does not know where you are at the moment.</source>
+        <translation>
+            <numerusform>You have a map loaded (1 room), but Mudlet does not know where you are at the moment.</numerusform>
+            <numerusform>You have a map loaded (%n rooms), but Mudlet does not know where you are at the moment.</numerusform>
         </translation>
     </message>
 </context>
 <context>
     <name>TRoomDB</name>
     <message numerus="yes">
-        <location filename="../../src/TRoomDB.cpp" line="726"/>
+        <location filename="../../src/TRoomDB.cpp" line="740"/>
+        <source>[ INFO ]  - The missing area(s) are now called:
+(ID) ==&gt; &quot;name&quot;</source>
+        <comment>Making use of %n to allow quantity dependent message form 8-) !</comment>
+        <translatorcomment>Perhaps should put the %n back in the source code</translatorcomment>
+        <translation>
+            <numerusform>[ INFO ]  - The missing area is now called:
+(ID) ==&gt; &quot;name&quot;</numerusform>
+            <numerusform>[ INFO ]  - The missing %n areas are now called:
+(ID) ==&gt; &quot;name&quot;</numerusform>
+        </translation>
+    </message>
+    <message numerus="yes">
+        <location filename="../../src/TRoomDB.cpp" line="724"/>
         <source>[ ALERT ] - %n area(s) detected as missing in map: adding it/them in.
- Look for further messsages related to the rooms that are supposed
+ Look for further messages related to the rooms that are supposed
```

Comment:
These are (probably my) typos that slipped into the source text and were cleaned up a long time ago - but the `mudlet_en_US.ts` file didn't get picked up upon - but now they have. So these changes are wanted.